core/src/main/java/io/grpc/internal/AbstractServerImplBuilder.java

```diff
@@ -71,6 +72,11 @@ protected AbstractServerImplBuilder() {
   }
 
   @Override
+  public final T directExecutor() {
+    return executor(MoreExecutors.directExecutor());
```

Comment:
I am using `directExecutor()` and not `newDirectExecutorService()`. Unsure why `newDirectExecutorService()` has been used so far mostly?